src/Screens/Phpunit.php

```diff
@@ -18,13 +18,16 @@ class Phpunit extends Screen
     /** @var string */
     private $phpunitBinaryPath;
 
+    /** @var integer */
+    private $phpunitTimeout;
+
     public function __construct(array $options)
     {
         $this->options = $options;
 
         $this->phpunitArguments = $options['phpunit']['arguments'] ?? '';
-
         $this->phpunitBinaryPath = $options['phpunit']['binaryPath'] ?? self::DEFAULT_BINARY_PATH;
+        $this->phpunitTimeout = $options['phpunit']['timeout'] ?? 60;
```

Comment:
Is `60` the default being used if `setTimeout` is not called on `Process`?

Reply:
Yes, it is.